libs/core/src/lib/time-picker/time-picker.component.ts

```diff
@@ -131,11 +144,33 @@ export class TimePickerComponent implements ControlValueAccessor, OnInit {
     /** @hidden */
     onTouched: Function = () => {};
 
+    /** @hidden */
+    constructor(private _cd: ChangeDetectorRef, public timeAdapter: TimeFormatParser) {}
```

Comment:
just curious, why is timeAdapter public?

Reply:
It's leftover, previously it was used in template I guess. Changed to private